PurchasesTests/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>BNDL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>3.0.2</string>
```

Comment:
fastlane uses apple's agvtool under the hood, and it auto-bumps the tests' info.plist. I don't think it's necessarily a bad thing. Thoughts?

Reply:
I think it's fine :)